src/index.js

```diff
@@ -10,10 +10,23 @@ const addEventsUrl = baseUrl + '/addEvents';
 const uploadLogsUrl = baseUrl + '/api/uploadLogs';
 
 let logGroupOptions = {};
-if (process.env['LOG_GROUP_OPTIONS']) logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
-let defaultParserName = process.env['PARSER_NAME'];
+let userLogFile;
+let userParserName;
+let userServerHost;
+if (process.env['LOG_GROUP_OPTIONS']) {
+    logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
+    let functionLogGroupName = process.env['AWS_LAMBDA_LOG_GROUP_NAME'];
```

Comment:
could (and should) probably use "const" instead of "let" here since the value of functionLogGroupName (once assigned) never changes